### PR TITLE
Update README example

### PR DESCRIPTION
**Pull request by Konstantine00**
This fixes the example in the README so that it will work out of the box. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +32,16 @@ This example demonstrates how to open a WebSocket URL:
...
 from trio_websocket import open_websocket_url
 
 
 async def main():
     try:
-        async with open_websocket_url('wss://localhost/foo') as ws:
+        async with open_websocket_url('ws://127.0.0.1:8000') as ws:
             await ws.send_message('hello world!')
             message = await ws.get_message()
+            print("Here's what came back: " + message)
```
Comment (belm0) [inline README.md]: To make the original logging.info() line below work, add this before main():
```
logging.root.setLevel(logging.INFO)
```
However in the interest of keeping the example simple and reducing dependencies, I'd rather just use print:

```
print('Received message: %s' % message)
```

```
from sys import stderr
...
print('Connection attempt failed: %s' % ose, file=stderr)
```

@mehaase

File: README.md [hunk 2]
```diff
@@ -32,14 +32,16 @@ This example demonstrates how to open a WebSocket URL:
...
 ```python
 import trio
+import logging
 from trio_websocket import open_websocket_url
 
 
 async def main():
     try:
-        async with open_websocket_url('wss://localhost/foo') as ws:
+        async with open_websocket_url('ws://127.0.0.1:8000') as ws:
```
Comment (belm0) [inline README.md]: This endpoint is just as likely not to work for anyone running the demo.  I'd prefer using a dedicated example domain.  Use of "wss" and path emphasize SSL and path support of the library.
```
open_websocket_url('wss://example.com/ws_path')
```

Comment (Konstantine00) [reply README.md]: yea that seems to work well :)

**Comment by Konstantine00** [comment]
How's this?

File: README.md (hunk 1 above)
Comment (Konstantine00) [reply README.md]: I like that idea

**Pull request merged**